### PR TITLE
support set model_filename and params_filename in post_training_quantization, test=develop

### PR DESCRIPTION
**Pull request by juncaipeng**
* support set model_filename and params_filename in post_training_quantization
* modify the inputs name

### CODE REVIEW

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py [hunk 1]
```diff
@@ -34,8 +34,10 @@
 class PostTrainingQuantization(object):
     def __init__(self,
                  executor,
-                 model_path,
-                 data_reader,
+                 sample_generator,
+                 model_params_path,
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py]: `model_params_path` --> `model_dir`或者`model_dirname`?

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py [hunk 2]
```diff
@@ -79,18 +90,29 @@ def __init__(self,
...
-            model_path = load_fp32_model_path
-            save_model_path = save_int8_path
-            data_reader =  your_data_reader
+            model_params_path = path/to/fp32_model_params
+            # set model_filename as None when the filename is __model__, 
+            # otherwise set it as the real filename
+            model_filename = None 
+            # set params_filename as None when all parameters were saved in 
+            # separate files, otherwise set it as the real filename
+            params_filename = None
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py:99]: `model_params_path`注意改一下。`model_filename `和`params_filename `可以注明默认是None，所以在此也可不设置。

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py [hunk 3]
```diff
@@ -79,18 +90,29 @@ def __init__(self,
...
             algo = "KL"
-            quantizable_op_type = ["conv2d", \
-                "depthwise_conv2d", "mul", "pool2d", "elementwise_add"]
+            quantizable_op_type = ["conv2d", "depthwise_conv2d", "mul"]
             ptq = PostTrainingQuantization(
                         executor=exe,
-                        model_path=model_path,
-                        data_reader=data_reader,
+                        sample_generator=sample_generator,
+                        model_params_path=model_params_path,
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py]: `model_params_path `这里同样需要改一下。

**Review by wzzju — CHANGES_REQUESTED** [review]

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py (hunk 2 above)
Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py:99]: Done

Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py:99]: model_filename 和params_filename说明白，免得大家需要仔细查看文档，因为大部分模型的权重都是保存成一个文件。

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py [hunk 4]
```diff
@@ -79,18 +90,29 @@ def __init__(self,
...
+            model_dir = path/to/fp32_model_params
+            # set model_filename as None when the filename is __model__, 
+            # otherwise set it as the real filename
+            model_filename = None 
+            # set params_filename as None when all parameters were saved in 
+            # separate files, otherwise set it as the real filename
+            params_filename = None
+            save_model_path = path/to/save_model_path
+            # prepare the sample generator according to the model, and the 
+            # sample generator must return a simple every time. The reference
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py:102]: `return a simple every time` --> `return one sample every time`

**Review by wzzju — APPROVED** [review]
Mostly LGTM.

**Pull request merged**